### ISSUE TITLE
Vignette notation discrepancy

### ISSUE THREAD

**Issue by yunqiyang0215** [stephenslab/udr#19]
`ud_init() ` takes `dat` as input, and `dat` can either from `mash_set_data()` or `X`. But `X` is not mentioned before using it. Same problem occurs in 
```
U_scaled = list(indep = diag(ncol(X)), equal = matrix(1, ncol(X), ncol(X)) + 0.0001 *
    diag(ncol(X)))
```
See here: https://stephenslab.github.io/udr/reference/ud_init.html

**Comment by pcarbo** [comment]
@yunqiyang0215 Can you please test the new version of ud_init with a data matrix and a mash data object to make sure it works with both those data types?

**Comment by yunqiyang0215** [comment]
Hi @pcarbo , I just tested the implementation and it works. But I think the documentation about the function `ud_init()` also needs to be updated.

**Comment by pcarbo** [comment]
Better now?

**Comment by yunqiyang0215** [comment]
Yes. A minor point, for the explanation of `dat`. 

`An n x m data matrix, in which each row of the matrix is an m-dimensional data point, or a “mash” object, for example created by [mash_set_data](https://rdrr.io/pkg/mashr/man/mash_set_data.html). When X is a matrix it should have at least 2 rows and at least 2 columns`. 

We should also replace `X` with `dat`?

**Comment by pcarbo** [comment]
Yes, good catch. This should now be fixed. I didn't update the pkgdown webpage with the change, but I'm sure we will have another opportunity to do so soon.

**Comment by yunqiyang0215** [comment]
I see. Sounds good. Thanks, Peter!
